app-ios/Modules/Sources/Timetable/TimetableListItemView.swift

```diff
@@ -69,8 +84,22 @@ struct TimetableListItemView: View {
     }
 }
 
- #Preview {
-     TimetableListItemView(
-         timetableItemWithFavorite: TimetableItemWithFavorite.companion.fake()
-     )
- }
+private extension RoomType {
```

Comment:
RoomIndex has been removed in https://github.com/DroidKaigi/conference-app-2023/pull/665, so replaced it with RoomType https://github.com/DroidKaigi/conference-app-2023/pull/724/commits/675728e0fa41ba275dc59735c67724341a4bc1a7

Reply:
@ry-itto 
On my local, Xcode says that RoomType has no member roomA 😵‍💫
Could you double check if there is actually no member or not? 🙏
![image](https://github.com/DroidKaigi/conference-app-2023/assets/43767445/8a774daf-4ea4-4d2e-a887-43e61f6d921b)

Reply:
These cases seems to renam to `rooma` `roomb` , etc...
Can you rename it? 🙏🏼

Reply:
Oh... it worked by renaming them... https://github.com/DroidKaigi/conference-app-2023/pull/724/commits/7dae6d6d5ad2cb4520ac8fdaa4625f6ad214cb3d
Xcode didn't suggest me at all and I didn't know that KMP could not consider camel cases... 😇

Thank you for your help!